app/src/main/java/com/spoony/spoony/core/util/extension/ColorExt.kt

```diff
@@ -0,0 +1,8 @@
+package com.spoony.spoony.core.util.extension
+
+import androidx.compose.ui.graphics.Color
+
+fun Color.Companion.hexToColor(hex: String): Color {
+    val colorInt = android.graphics.Color.parseColor("#$hex")
```

Comment:
p4) 위에 import있는데 지워도 되지 않을까요?

```suggestion
    val colorInt = Color.parseColor("#$hex")
```

Reply:
지웠습니다..!